VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -4,8 +4,8 @@
     android:layout_width="match_parent"
     android:layout_height="64dp"
     android:paddingTop="4dp"
-    android:paddingBottom="4dp"
     android:orientation="vertical"
+    android:background="@drawable/course_list_item_selector"
```

Comment:
OK, so we can't use `listSelector` because our `CourseBaseAdapter` structure is designed to capture item clicks, with the expectation that the Activities and Fragments that use it would create an anonymous extension of it to override the `rowClicked()` and `download()` callbacks. This results in the click events being consumed before being propagated to the `ListView` level. Perhaps we should remove this anti-pattern in favor of the more sane callback interface design.